app/main.py

```diff
@@ -1,3 +1,49 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: int | float | Distance) -> Distance:
+        if isinstance(other, Distance):
+            return Distance(self.km + other.km)
+        elif isinstance(other, int | float):
+            return Distance(self.km + other)
+
+    def __iadd__(self, other: int | float | Distance) -> Distance:
+        if isinstance(other, Distance):
+            self.km += other.km
+            return self
+        elif isinstance(other, int | float | Distance):
+            self.km += other
+            return self
+
+    def __mul__(self, other: int | float | Distance) -> Distance:
+        mul_dist = self.km * other
+        return Distance(mul_dist)
+
+    def __truediv__(self, other: int | float | Distance) -> Distance:
+        div_dist = round(self.km / other, 2)
+        return Distance(div_dist)
+
+    def __lt__(self, other: int | float | Distance) -> bool:
+        return self.km < other
```

Comment:
If `other` can be Distance, you should handler this option inside function. Fix everywhere
```suggestion
    def __lt__(self, other: int | float | Distance) -> bool:
        if isinstance(other, Distance):
              return self.km < other.km
        return self.km < other
```